DFC.ServiceTaxonomy.GraphSync/Services/TitlePartCloneGenerator.cs

```diff
@@ -0,0 +1,12 @@
+﻿using DFC.ServiceTaxonomy.GraphSync.Services.Interface;
+
+namespace DFC.ServiceTaxonomy.GraphSync.Services
+{
+    public class TitlePartCloneGenerator : ITitlePartCloneGenerator
+    {
+        public string Generate(string title)
+        {
+            return title.StartsWith("CLONE - ") ? title : $"CLONE - {title}";
```

Comment:
We should probably do something similar with the title as we do with the urls, so start adding a number on further clones.